frontend/src/pages/Workflows/WorkflowItem.js

```diff
@@ -449,15 +457,45 @@ const renderActionButtons = (
           data-test="show-workflowitem-permissions"
           iconButtonClassName={getButtonClass(workflowSortEnabled, status)}
         />
+        )
+
+        }
+
+        {
+         workflowSortEnabled || status === "closed" || closeDisabled ? null :(
+         <>
+           <ActionButton
+            notVisible={workflowSortEnabled || status === "closed" || closeDisabled}
+            onClick={closeDisabled ? undefined : reject}
```

Comment:
Here you have 2 conditions in line 465 and 468. Only use 1. Moreover, move the conditions to up to line 422 and give it a name that makes sense to see the business rule immediately

Reply:
Do this for the other vars in the file too